### PR TITLE
Hearing Worksheet : Persist Appellant/Veteran Information to redux

### PR DESCRIPTION
**Pull request by nolawi**
Connects #2474

### CODE REVIEW

File: client/app/hearings/HearingWorksheet.jsx [hunk 1]
```diff
@@ -210,8 +213,13 @@ export class HearingWorksheet extends React.PureComponent {
             <div>Veterans of Foreign Wars</div>
           </div>
           <div className="cf-hearings-worksheet-data-cell column-5">
-            <label htmlFor="appellant-vet-rep-name">Rep. Name:</label>
-            <input id="appellant-vet-rep-name" aria-label="Representative Name" type="text" />
+            <TextField
+              name="Rep. Name:"
+              id="appellant-vet-rep-name"
+              aria-label="Representative Name"
+              value={this.props.worksheet.repName || ''}
```
Comment (joofsh) [inline client/app/hearings/HearingWorksheet.jsx]: similar to the last PR, if we have to do this everywhere then maybe the component internally should handle a `null` value. We could update the `TextField` component to support `null` being passed in, that way we don't have to do this each time it's used

File: client/test/karma/hearings/reducers/index-test.js [hunk 2]
```diff
@@ -19,6 +20,41 @@ describe('hearingsReducer', () => {
...
   });
 
+  context(Constants.SET_REPNAME, () => {
+    let state;
+
+    beforeEach(() => {
+      state = Hearings.hearingsReducers(initialState, {
+        type: Constants.SET_REPNAME,
+        payload: {
+          repName: 'filled'
```
Comment (joofsh) [inline client/test/karma/hearings/reducers/index-test.js]: minor nit: to help future developers with context do you mind adding a "real" value here. A fake name "Jane Smith" makes it more clear how the field will be used in the future than "filled"

Comment (nolawi) [reply client/test/karma/hearings/reducers/index-test.js]: for reference /issues/3108

File: client/app/hearings/HearingWorksheet.jsx (hunk 1 above)
Comment (nolawi) [reply client/app/hearings/HearingWorksheet.jsx]: for reference /issues/3108

**Pull request merged**